front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -19,22 +18,76 @@
     }
   }
 
+  .micro-macro-buttons {
+    button {
+      color: var(--grey60);
+    }
+    font-size: 0.875rem;
+    letter-spacing: 0.1019rem;
+    line-height: 1.125rem;
+    margin-block: 1.813rem 2.063rem;
+    padding-block: 0.25rem;
+    display: inline-flex;
+    background-color: var(--ambiantB10);
+    border-radius: 0.375rem;
+    box-shadow:
+      0 0.125rem 0.125rem -0.0625rem rgba(255, 171, 88, 0.27),
+      0 0.0625rem 0.125rem rgba(0, 0, 0, 0.16);
+    .micro-button {
+      border-right: solid 0.0313rem var(--grey20);
+      padding: 0.188rem 0.625rem 0.188rem 1rem;
+    }
+    .macro-button {
+      border-left: solid 0.0313rem var(--grey20);
+      padding: 0.188rem 1rem 0.188rem 0.625rem;
+    }
+    .micro-button:hover {
+      box-shadow:
+        0 0.25rem 0rem var(--white100),
+        0 -0.25rem 0rem var(--white100);
+      border-radius: 0.375rem 0 0 0.375rem;
+      color: var(--primary80);
+    }
+    .macro-button:hover {
+      box-shadow:
+        0 0.25rem 0rem var(--white100),
+        0 -0.25rem 0rem var(--white100);
+      border-radius: 0 0.375rem 0.375rem 0;
+      color: var(--primary80);
+    }
+    .micro-button:active {
+      color: var(--grey30);
```

Comment:
`grey30` var is not added yet in `_variables.scss`

Reply:
Done